tests/__init.test.js

```diff
@@ -19,13 +19,14 @@ navigator.getVRDisplays = function () {
   return Promise.resolve([mockVRDisplay]);
 };
 
-require('index');
+const AFRAME = require('index');
```

Comment:
Sorry for the `const` here :) you can modify it to `var` directly on master if you want.